Formula/platformio.rb

```diff
@@ -13,7 +13,7 @@ class Platformio < Formula
     sha256 "c6501bd561ee8d2b77037739484929eade3c4e15cb326b24b90634b32daecd32" => :high_sierra
   end
 
-  depends_on "python@3.8"
+  depends_on "python@3"
```

Comment:
If it doesn't support python 3.8 this should just be `python`

Reply:
It supports Python 3.5+ including Python 3.8. I don't understand why we should install Python 3.8 hen Python 3.7 (`python@3`) is already installed? The same on my Mac machine: some tools depenend on Python 3 and some on Python 3.8. Why do I need multiple same Pythons?

Reply:
Is 84 line "virtualenv_create(libexec, "python3")" ok? The ` homebrew-pypi-poet`added it.

Reply:
> some tools depenend on Python 3 and some on Python 3.8. Why do I need multiple same Pythons?

Yes, everything that works with 3.8 will depend on that version. As soon as everything works with 3.8 you can remove 3.7.

Reply:
Could you merge this?